Content.Server/Body/Systems/BloodstreamSystem.cs

```diff
@@ -39,7 +39,7 @@ public override void Initialize()
     {
         base.Initialize();
 
-        SubscribeLocalEvent<BloodstreamComponent, ComponentInit>(OnComponentInit);
```

Comment:
Solution Manager is not ready on Init so I moved this to Startup instead. I don't know the global repercussions of this change, please advise.

Reply:
init is raised when that component is added, but not before other components are initialized. Startup is called once all the components are initialized. In short, startup is probably the way to go here.